freelist.go

```diff
@@ -171,7 +172,9 @@ func (f *freelist) read(p *page) {
 	copy(f.ids, ids)
 
 	// Make sure they're sorted.
-	sort.Sort(pgids(f.ids))
+	if !pgids(f.ids).isSorted() {
```

Comment:
Look, I was investigating Bolt more than year ago. I need it to open many mostly-read-only files fast. That is why I concentrated on open. All patches were benched, but I have no numbers now. Neither I use Bolt now, cause my director didn't ratify my suggestion of using it.
So now I don't care will this pull request be accepted or not.
If you benchmark it and find it will help you, then it will be your deal to push on merging this pull request.
Though, if you find some error here, I will fix it.